jobs/config.json

```diff
@@ -3254,7 +3254,7 @@
       "--gcp-zone=us-east1-a",
       "--ginkgo-parallel=30",
       "--provider=gce",
-      "--test_args=--ginkgo.skip=\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]|\\[DisabledForLargeClusters\\] --allowed-not-ready-nodes=50 --node-schedulable-timeout=90m --minStartupPods=8 --gather-resource-usage=master --gather-metrics-at-teardown=master",
+      "--test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]|\\[DisabledForLargeClusters\\] --allowed-not-ready-nodes=50 --node-schedulable-timeout=90m --minStartupPods=8 --gather-resource-usage=master --gather-metrics-at-teardown=master",
```

Comment:
You could instead set 'GINKGO_TOLERATE_FLAKES=y' in the job's env file.
We're doing that for e2e tests on our PR job. IIRC the effect is the same though.

Reply:
Also could you the make the change for gce-large-correctness?

Reply:
Done. I prefer this option rather than the env var, since this makes the number of flake attempts explicit.